.github/workflows/ci-angular.yaml

```diff
@@ -0,0 +1,80 @@
+name: Angular CI
+
+on:
+  push:
+    branches: [ "master" ]
+env:
+  IMAGE_NAME: angular
+  GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+jobs:
+  testing:
+     runs-on: ubuntu-latest
+     outputs:
+        get-job-id-test: ${{steps.job-id-test.outputs.get-job-id-test-out}}
+        short-sha1: ${{steps.env-short-sha.outputs.short-sha}}
+     strategy:
+         fail-fast: true
+     steps:
+     - name: Take Short SHA
+       id: env-short-sha
+       run: echo "::set-output name=short-sha::${GITHUB_SHA::7}"
+     - uses: actions/checkout@v1
+     - name: Use Node.js
```

Comment:
pls use semantic short names and always use lower case

Reply:
Fixed